### ISSUE TITLE
Blacklist mob?

### ISSUE THREAD

**Issue by Sunekaer** [Malorolam/LootBags#150]
Hey 

Is it possible to blacklist an mob so it doesn’t drop a loot bag ?

https://github.com/Sunekaer/stoneBlock/issues/88

**Comment by Malorolam** [comment]
Yes, $EXCLUDEENTITIES:<Boolean> when set to false means the entity list for that bag acts as a blacklist.
